biocypher_metta/adapters/ontologies_adapter.py

```diff
@@ -1,168 +1,144 @@
-        # 'orphanet': 'https://www.orphadata.com/data/ontologies/ordo/last_version/ORDO_en_4.3.owl',
-        # 'ncit': 'http://purl.obolibrary.org/obo/ncit.owl'
-    }
-
-    GO_SUBONTOLGIES = ['molecular_function',
-                       'cellular_component', 'biological_process']
+    ONTOLOGIES = {}
 
     HAS_PART = rdflib.term.URIRef('http://purl.obolibrary.org/obo/BFO_0000051')
     PART_OF = rdflib.term.URIRef('http://purl.obolibrary.org/obo/BFO_0000050')
-    SUBCLASS = rdflib.term.URIRef(
-        'http://www.w3.org/2000/01/rdf-schema#subClassOf')
-    DB_XREF = rdflib.term.URIRef(
-        'http://www.geneontology.org/formats/oboInOwl#hasDbXref')
+    SUBCLASS = rdflib.term.URIRef('http://www.w3.org/2000/01/rdf-schema#subClassOf')
+    DB_XREF = rdflib.term.URIRef('http://www.geneontology.org/formats/oboInOwl#hasDbXref')
 
     LABEL = rdflib.term.URIRef('http://www.w3.org/2000/01/rdf-schema#label')
-    RESTRICTION = rdflib.term.URIRef(
-        'http://www.w3.org/2002/07/owl#Restriction')
-    TYPE = rdflib.term.URIRef(
-        'http://www.w3.org/1999/02/22-rdf-syntax-ns#type')
-    ON_PROPERTY = rdflib.term.URIRef(
-        'http://www.w3.org/2002/07/owl#onProperty')
-    SOME_VALUES_FROM = rdflib.term.URIRef(
-        'http://www.w3.org/2002/07/owl#someValuesFrom')
-    ALL_VALUES_FROM = rdflib.term.URIRef(
-        'http://www.w3.org/2002/07/owl#allValuesFrom')
-    NAMESPACE = rdflib.term.URIRef(
-        'http://www.geneontology.org/formats/oboInOwl#hasOBONamespace')
-    EXACT_SYNONYM = rdflib.term.URIRef(
-        'http://www.geneontology.org/formats/oboInOwl#hasExactSynonym')
-    RELATED_SYNONYM = rdflib.term.URIRef(
-        'http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym')
-    DESCRIPTION = rdflib.term.URIRef(
-        'http://purl.obolibrary.org/obo/IAO_0000115')
+    RESTRICTION = rdflib.term.URIRef('http://www.w3.org/2002/07/owl#Restriction')
+    TYPE = rdflib.term.URIRef('http://www.w3.org/1999/02/22-rdf-syntax-ns#type')
+    ON_PROPERTY = rdflib.term.URIRef('http://www.w3.org/2002/07/owl#onProperty')
+    SOME_VALUES_FROM = rdflib.term.URIRef('http://www.w3.org/2002/07/owl#someValuesFrom')
+    ALL_VALUES_FROM = rdflib.term.URIRef('http://www.w3.org/2002/07/owl#allValuesFrom')
+    NAMESPACE = rdflib.term.URIRef('http://www.geneontology.org/formats/oboInOwl#hasOBONamespace')
+    EXACT_SYNONYM = rdflib.term.URIRef('http://www.geneontology.org/formats/oboInOwl#hasExactSynonym')
+    RELATED_SYNONYM = rdflib.term.URIRef('http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym')
+    DESCRIPTION = rdflib.term.URIRef('http://purl.obolibrary.org/obo/IAO_0000115')
 
     PREDICATES = [SUBCLASS, DB_XREF]
     RESTRICTION_PREDICATES = [HAS_PART, PART_OF]
 
-    def __init__(self, type='node', dry_run=False):
+    def __init__(self, write_properties, add_provenance, ontology, type, label, dry_run=False):
+        super().__init__(write_properties, add_provenance)
```

Comment:
Minor but better to pass arguments to the parent class after instantiating the fields of the current class